src/systems/ec/specs/components/inpage-navigation/docs/usage.md

```diff
@@ -1 +1,15 @@
-Do's and don'ts
+The in-page navigation is a vertical menu containing (anchored) links to H2-headings on long content pages. It appears at the left side of a page. It gives the user an overview of the page contents and allows them to jump directly to content that is lower on the page without having to scroll through the rest of the page.  Most content pages on ec.europa.eu make use of the in-page navigation component.
+
+A page with in-page navigation also makes it easier to refer to specific content under an H2 header when that page is shared.
+
+<img src="https://inno-ecl.s3.amazonaws.com/media/images/EC/Inpage/inpage.png" alt="in-page navigation" width="920"/>
+
+## When to use
+
+* Use on pages with extensive body text that is clearly structured with H2 headings
+* It is a floating menu, meaning it is always at the top of the page, in the left column
+* Note that the page’s left column is reserved for the in-page navigation
```

Comment:
does it mean that https://ec.europa.eu/info/business-economy-euro/doing-business-eu/contract-rules/insurance-contracts/what-commission-has-been-doing-insurance-contracts_en should have left a blank column on the left?